dydx/chain.json

```diff
@@ -59,7 +59,7 @@
     },
     "versions": [
       {
-        "name": "v2",
+        "name": "v2.0",
```

Comment:
"name" has to match the upgrade_handler - in this case it's fine since the chain started with v2 - but for later modifications you made (i.e. v3.0.0 to v3.0) are not valid as the upgrade_handler is v3.0.0.

Reply:
got it!